Examples/VeraSDKExample-CP/VeraSDKExample-CP/GoogleService-Info.plist

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>345454492227-arss1ed6sf31b4d174d0qmjhdkbh1bvl.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.345454492227-arss1ed6sf31b4d174d0qmjhdkbh1bvl</string>
+	<key>ANDROID_CLIENT_ID</key>
+	<string>345454492227-0p2lbv2adtk8n9ebcjgltuebcgh51vgt.apps.googleusercontent.com</string>
+	<key>API_KEY</key>
+	<string>AIzaSyAIg9tthuryOiuS76Cgk9gOYpPVQg-Z7Bk</string>
```

Comment:
@eyal-resonai is it ok to expose this?

Reply:
I don't know - where did you get it from?

Reply:
@eyal-resonai this is the only question left. This file is needed for Firebase SDK to work, it's from a demo project I created in Firebase under the ResonAI account. This API key however is the same for all our apps from what I see, and I don't know if it should be made so public.